### PR TITLE
Adjustments to elements and migrations

### PR DESCRIPTION
**Pull request by andreytroeglazov**
As discussed:

1. Added body field to CTA
2. Added `buildParagraphTitle` method
3. Adjusted images migration
4. Added bottom padding to Landing page
5. Removed bottom padding from cards template

### CODE REVIEW

**Comment by andreytroeglazov** [comment]
screencapture-drupal-starter-ddev-site-4443-2023-02-02-14_29_34

File: web/modules/custom/server_general/src/Plugin/EntityViewBuilder/NodeLandingPage.php [hunk 1]
```diff
@@ -42,7 +42,8 @@ public function buildFull(array $build, NodeInterface $entity) {
     $element = $this->buildReferencedEntities($paragraphs);
     $elements[] = $this->wrapContainerVerticalSpacingBig($element);
 
-    $build[] = $this->wrapContainerVerticalSpacingBig($elements);
+    $elements = $this->wrapContainerVerticalSpacingBig($elements);
+    $build[] = $this->wrapContainerBottomPadding($elements);
```
Comment (amitaibu) [inline web/modules/custom/server_general/src/Plugin/EntityViewBuilder/NodeLandingPage.php:46]:
```suggestion
    // Add bottom padding, so there's some padding between all the paragraphs
    // and the footer.
    $build[] = $this->wrapContainerBottomPadding($elements);
```

**Review by amitaibu — APPROVED** [review]
Minor comment and MWG

File: web/modules/custom/server_general/src/Plugin/EntityViewBuilder/NodeLandingPage.php (hunk 1 above)
Comment (andreytroeglazov) [reply web/modules/custom/server_general/src/Plugin/EntityViewBuilder/NodeLandingPage.php:48]: :+1:

**Pull request merged**